src/frontend/screens/Settings/ProjectSettings/YourTeam/SelectDevice.tsx

```diff
@@ -73,12 +91,9 @@ function InvitableDevicesList() {
   return (
     <View style={styles.deviceListContainer}>
       {invitableDevices.map(device => {
-        const {deviceId, status, name} = device;
-
-        // TODO: Use `device.deviceType`
-        const deviceType = 'mobile';
+        const {deviceId, status, name, deviceType: rawDeviceType} = device;
+        const deviceType = normalizeDeviceType(rawDeviceType);
```

Comment:
think it'd be clearer to do the narrowing where it's needed in the UI as opposed to here.